src/Atmos/Model/moisture.jl

```diff
@@ -40,11 +43,12 @@ Assumes the moisture components is in the dry limit.
 struct DryModel <: MoistureModel
 end
 
-vars_aux(::DryModel,T) = @vars(e_int::T)
+vars_aux(::DryModel,T) = @vars(e_int::T, temperature::T, R_m::T)
```

Comment:
Can we remove `R_m` from the aux state? I think we should just compute this locally using `R_m(thermo_state(...))`.